docs/user/advanced.rst

```diff
@@ -64,26 +64,21 @@ Some applications render templates based on data sources (e.g. CSVs or
 JSON files).
 
 To get data to templates you can set up a mapping between filenames
-and functions which generate dictionaries containing the data:
+and dictionaries or functions which generate dictionaries containing 
+the data:
 
 .. code-block:: python
 
     from staticjinja import make_site
 
-    def get_knights():
-        """Generate knights of the round table."""
-        knights = [
-            'sir arthur',
-            'sir lancelot',
-            'sir galahad',
-        ]
-        return {'knights': knights}
-
     if __name__ == "__main__":
         site = make_site(contexts=[
-            ('index.html', get_knights),
+            (
+             'index.html', 
+             { 'knights': ['sir arthur', 'sir lancelot', 'sir galahad'] }
```

Comment:
Having some doubts whether or not allowing this is a good idea after thinking more over it. I'm just worried that by adding another way of doing things we're making more complex. Presently, one could get this behavior just using `lambda`.